packages/connector-azuread/README.md

```diff
@@ -0,0 +1,57 @@
+# Azure AD connector
+
+The Azure AD connector provides a succinct way for your application to use Azure’s OAuth 2.0 authentication system.
+
+**Table of contents**
+- [Azure AD connector](#azureAD-connector)
+  - [Set up Azure AD in the Azure Portal](#set-up-azure-ad-in-the-azure-portal)
+  - [Configure your client secret](#configure-your-client-secret)
+  - [Compose the connector JSON](#compose-the-connector-json)
+    - [Config types](#config-types)
+  - [References](#references)
+
+## Set up Azure AD in the Azure Portal
+
+- Visit the [Azure Portal](https://portal.azure.com/#home) and sign in with your Azure account. You need to have an active subscription to access Azure AD.
+- Click the **Azure Active Directory** from the services they offer, and click the **App Registrations** from the left menu.
+- Click **New Registration** at the top and enter a description, select your **access type** and add your **Redirect URI**, which redirect the user to the application after logging in. In our case, this will be `${your_logto_origin}/callback/azuread-universal`. e.g. `https://logto.dev/callback/azuread-universal`.
+- If you select **Single Tennant** for access type than you need to enter **TennantID**, else you need to enter `common` as tennantID
+
+
+## Configure your client secret
+- In your newly created project, click the **Certificates & Secrets** to get a client secret, and click the **New client secret** from the top.
+- Enter a description and an expiration.
+- This will only show your client secret once. Save the **value** to a secure location.
+
+## Compose the connector JSON
+- Add your App Registration's **Client ID** into logto json.
+- Add your **Client Secret** into logto json.
+- Add your App Registration's **Tennant ID** into logto json.
+- Add your Microsoft **Login Url** into logto json. This defaults to "https://login.microsoftonline.com/" for many applications, but you can set your custom domain if you have one. (Don't forget the trailing slash)
+
+
+```json
+{
+  "clientId": "<client-id>",
+  "clientSecret": "<client-secret>",
+  "tennantId": "<tennant-id>",
+  "cloudInstance": "https://login.microsoftonline.com/"
+}
+```
+
+> ℹ️ **Note**
+> 
+> Users need to have `DisplayName` and `Mail` fields filled inside the Azure Active Directory in order to register successfully into Logto.
```

Comment:
i think we can allow empty display name and mail for now since we can still get the unique `id` of the user. in the future, we're likely to make it configurable about what scope to require for a connector (e.g. email)